playbook/app/pb_kits/playbook/pb_table/styles/_sticky_header.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../tokens/colors";
+
+[class^="pb_table"] {
+  &.sticky-header {
+    thead {
+      background: $white;
+      position: sticky;
+      top: -40px;
+      z-index: 10;
```

Comment:
Why 10?

Reply:
On google searches I saw people using 10 it worked, but Im okay with a different suggestion and learning why I should or should not use 10 here.

Reply:
I'd like to keep it low. z-index is a really tricky thing to deal with. I think setting to match the rich text editor sticky is the way to go.

Reply:
[Use this as an example](https://github.com/powerhome/playbook/blob/PLAY-67-add-sticky-headers-for-table-kit/playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss#L84)